core/logger/default.go

```diff
@@ -22,28 +22,11 @@ func init() {
 		log.Fatalf("failed to register os specific sinks %+v", err)
 	}
 
-	// HACK: This logic is a bit duplicated from newProductionConfig but the config object is not available in init
-	// To be removed with https://app.shortcut.com/chainlinklabs/story/18500/logger-injection
-	jsonStr := os.Getenv("JSON_CONSOLE")
-	var jsonConsole bool
-	if jsonStr == "true" {
-		jsonConsole = true
-	}
-	unixTSStr := os.Getenv("LOG_UNIX_TS")
-	var unixTS bool
-	if unixTSStr == "true" {
-		unixTS = true
-	}
-	toDiskStr := os.Getenv("LOG_TO_DISK")
-	var toDisk bool
-	if toDiskStr == "true" {
-		toDisk = true
-	}
+	jsonConsole := os.Getenv("JSON_CONSOLE") == "true"
```

Comment:
Nitpick, but maybe worth using `strconv.ParseBool` for a little more flexibility?